plugins/chat/app/services/chat/auto_leave_channels.rb

```diff
@@ -34,13 +36,15 @@ def remove_memberships(params:)
 
       if !group_ids.include?(Group::AUTO_GROUPS[:everyone])
         sql = <<~SQL
+          -- event = #{params.event}
```

Comment:
This will hopefully help facilitate identifying which event triggers these queries in production in case they end up RANK 1 of most resource intensive queries in production